include/secure_services.h

```diff
@@ -15,6 +18,24 @@ extern "C" {
  */
 void spm_request_system_reboot(void);
 
+
+/** Request a random number from the Secure Firmware.
+ *
+ * This provides a True Random Number from the on-board random number generator
+ *
+ * @note Currently, the RNG hardware is run each time this is called. This
+ *       spends significant time and power.
+ *
+ * @param[out] output  The random number. Must be at least len long.
+ * @param[in]  len     The length of the output array.
```

Comment:
Update to conform to current requirements: Must be 144 long

Reply:
Added sentence